### PR TITLE
Feature: add placeholder text for no search result

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Add extension for UICollectionView to display placeholder text.

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -81,6 +81,7 @@
 
 "peoplepicker.no_matching_results_title" = "No results.";
 "peoplepicker.no_matching_results_message" = "Enter a full email address.";
+"peoplepicker.no_search results" = "No matching results. Try entering a different name.";
```
Comment (billypchan) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings]:
```suggestion
"peoplepicker.no_search_results" = "No matching results. Try entering a different name.";```

File: Wire-iOS/Sources/UserInterface/GroupDetails/GroupParticipantsDetail/GroupParticipantsDetailViewController.swift [hunk 2]
```diff
@@ -128,6 +132,9 @@ final class GroupParticipantsDetailViewController: UIViewController {
     private func participantsDidChange() {
         collectionViewController.sections = computeSections()
         collectionViewController.collectionView?.reloadData()
+        
+        let emptyResultMessage = (viewModel.admins.isEmpty && viewModel.members.isEmpty) ? "peoplepicker.no_search results".localized() : ""
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/GroupDetails/GroupParticipantsDetail/GroupParticipantsDetailViewController.swift]: need rename the localization key

**Comment by billypchan** [comment]
New snapshot test is needed or we can just update existing tests?

File: Wire-iOS/Sources/UserInterface/GroupDetails/GroupParticipantsDetail/GroupParticipantsDetailViewController.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/GroupDetails/GroupParticipantsDetail/GroupParticipantsDetailViewController.swift]: Fixed

File: Wire-iOS/Resources/Base.lproj/Localizable.strings (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: Fixed

**Comment by KaterinaWire** [comment]
> New snapshot test is needed or we can just update existing tests?

I added a new test and snapshot.

**Pull request merged**